cmd/lotus-storage-miner/market.go

```diff
@@ -191,8 +191,15 @@ var setAskCmd = &cli.Command{
 		}
 		defer closer()
 
-		pri := types.NewInt(cctx.Uint64("price"))
-		vpri := types.NewInt(cctx.Uint64("verified-price"))
+		pri, err := types.ParseFIL(cctx.String("price"))
+		if err != nil {
+			return err
+		}
+
+		vpri, err := types.ParseFIL(cctx.String("verified-price"))
+		if err != nil {
+			return err
+		}
```

Comment:
This change doesn't make sense given the reported issue. The prior code would produce a BigInt value of `1` for an import from `--price 1`. This change will produce a BigInt of `1000000000000000000` for the same input.

What was the reported behaviour prior?

Reply:
@travisperson Looks like you're right -- sounds like the fix should just be the helptext, then.